source/connect/mongoclient.txt

```diff
@@ -0,0 +1,132 @@
+.. _scala-mongoclient:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and performs 
+  operations on the MongoDB deployment.
```

Comment:
Perhaps the confusion was in the "performs operations" part since I want to say the user is performing the operations and not the object. Reworded, let me know what you think!